File: shoring/flywheel/src/test/java/com/panopset/tests/flywheel/DeepListTest.java
package com.panopset.tests.flywheel;

import java.io.IOException;
import org.junit.jupiter.api.Test;

public class DeepListTest {

  /**
   * Test using a list within a list.
   */
  @Test
  void testListCommand() throws IOException {
    SimpleTest.comparisonTest("deepListTest01.txt", "deepListTest.txt",
        "deepListTestExpected.txt");
  }

}

File: code/shoring/flywheel/src/test/kotlin/com/panopset/tests/flywheel/DeepListTest.kt
package com.panopset.tests.flywheel

import org.junit.jupiter.api.Test

class DeepListTest {
    /**
     * Test using a list within a list.
     */
    @Test
    fun testListCommand() {
        SimpleTest().comparisonTest(
            "deepListTest01.txt", "deepListTest.txt",
            "deepListTestExpected.txt"
        )
    }
}
